src/D7/FileDownloadTrait.php

```diff
@@ -318,4 +322,26 @@ protected function fileDownloadGetTempDir() {
     return '/tmp/behat_downloads';
   }
 
+  /**
+   * Before nodeCreate set field value for file.
+   *
+   * Field should be in format 'file;__file_source__;__file_name_.
+   *
+   * @beforeNodeCreate
+   */
+  public function nodeCreateAlter(EntityScope $scope) {
```

Comment:
this is not needed - the files are attached like so

```
 And article content:
      | title                | field_file                                                   |
      | [TEST] document page | example_file.txt |

```

and `example_file.txt` is located in `tests/fixtures`